app/controllers/app_controller.py

```diff
@@ -62,6 +56,13 @@ def __init__(self) -> None:
         self.main_window = MainWindow(settings_controller=self.settings_controller)
         self.main_window_controller = MainWindowController(self.main_window)
 
+        # Initialize themes
+        if self.settings.enable_themes is True:
+            self.app.setStyle("Fusion")
+            theme_name = self.settings.theme_name
+            theme = Themes(theme_name)
+            self.app.setStyleSheet(theme.style_sheet())
```

Comment:
How does this handle where the provided stylesheet is not a valid format?

Reply:
if the stylesheet in not valid format it is ignored there is a small validation at the moment which only allows .qss file

Reply:
What if it is an invalid qss file?

Reply:
its loaded but since its loaded on top of fusion it doesnt affect much